source/creating_content_with_studio/configuring_blocks/add-chart.rst

```diff
@@ -0,0 +1,86 @@
+Adding a chart
+==============
+
+You can use charts to create a graphic representation of your data.
+
+Step 1: Select the block type
+-----------------------------
+
+You need to define the block type before you configure the content you want to add.
+
+1. From the preview pane, click the block you want to edit.
+2. From the left side pane, select **Chart** as the block type.
+3. Click **Next**.
```

Comment:
The button is actually labeled "Configure".

Reply:
In this situation the button is "Next": this is when you create a new block from scratch. (well that's how I interpret the list anyway)

I may have missed something, or the text may have changed since your remark